datastore/src/main/java/com/ds/datastore/BookStoreController.java

```diff
@@ -71,11 +76,17 @@ private void registerWithHub() throws Exception {
         JsonObject json = new JsonObject();
         json.addProperty("id", this.id);
         json.addProperty("address", this.url + "/bookstores/" + this.id);
-        outputJson(con, gson, json);
+        String str = gson.toJson(json);
+        try(OutputStream os = con.getOutputStream()) {
+            byte[] input = str.getBytes(StandardCharsets.UTF_8);
+            os.write(input, 0, input.length);
+        }
+        int y = con.getResponseCode();
```

Comment:
do we do anything with this response code?

Reply:
If we are switching to java 9 then it doesn't matter anyways, no?

Reply:
true